### DISCUSSION TITLE
Shortcut Keys

### DISCUSSION THREAD

**Discussion by TLWise** [idaholab/moose#23343] [Q&A General] [answered]
Is there a listing somewhere of the shortcut key functions that are available in Ubuntu text editor? I am attempting to use the python Ctrl+/ method to comment out a line, but that is not working as expected.

**Comment by GiudGiud** [comment, marked as answer]
Hello

Which text editor?
We have a VSCode plugin that handles these things
https://mooseframework.inl.gov/help/development/VSCode.html
There's no extra list, it's more the classic VSCode shortcuts adapted to MOOSE inputs

Guillaume
